.gitignore

```diff
@@ -4,4 +4,7 @@ yarn-error.log
 npm-debug.log
 node_modules
 coverage
-webpack/webpack-assets.json
+webpack/
+.idea/
+generators/
+templates/
```

Comment:
I see that's something we need to do to better handle branch switching. It would be also very desirable in separate PRs.